internal/app/admin/http/cache.go

```diff
@@ -201,7 +201,11 @@ func cacheDumpHandler(o *orchestrator.Orchestrator) http.HandlerFunc {
 func clearCacheHandler(o *orchestrator.Orchestrator) http.HandlerFunc {
 	return func(w http.ResponseWriter, req *http.Request) {
 		if req.Method == http.MethodPost {
-			cacheKey := getParam(req.URL.Path)
+			var cacheKey string
```

Comment:
can we instead edit `getParam` https://github.com/envoyproxy/xds-relay/blob/8a16fd9860e4b65f51e8194e04ce22f16ead297b/internal/app/admin/http/handler.go#L157-L160 to return the last segment irrespective of length?

Reply:
Depends, I avoided doing this initially since we allowed defining `/cache` as equivalent to `/cache/` (i.e. print total cache). The log level handler also has meaning behind omitting the trailing slash. If we return final segment irrespective of length, this would cause a bug. Alternatively, we can require the final trailing slash for all handlers using `getParam` and this would work.

Reply:
how about using this https://github.com/envoyproxy/xds-relay/blob/eda7072ba1dee995cafa23e5f63d519cf63787c3/internal/app/admin/http/cache.go#L34

Reply:
I instead modified getParam() to take in a prefix to remove from the Path and used filepath.Split() to collect the remaining argument or empty string. I chose not to use Base() since it would return a `.` should the remaining path be empty.

Reply:
this approach lgtm. Can you make the urls constants so that there is less chance of bugs due to future changes.